doc/dev/developer_guide/index.rst

```diff
@@ -1586,6 +1586,94 @@ server list`` on the teuthology machine, but the target VM hostnames (e.g.
 ``target149202171058.teuthology``) are resolvable within the teuthology
 cluster.
 
+Runing Integration Tests using Teuthology
+=========================================
+
+Getting binaries
+----------------
+To run integration test using teuthology, you need to have Ceph binaries built
+for your branch. Following is how you can do it:
+
+#. Push your branch to `ceph-ci`_ repository. This would trigger the build
+   process.
+
+#. To confirm that the build process has initiated, spot your branch name
+   at `shaman`_. Little after the build process has initiated, the single
+   entry with your branch name would multiply, each new entry for a different
+   OS.
+
+#. Wait for building process to complete. Usually, it takes around 2-3 hours
+   depending on how busy the machines are.
+
+Triggering tests
+-----------------
+After building is complete are ready, proceed to trigger tests -
+
+#. Log in to the teuthology machine::
+
+       ssh user@teuthology.front.sepia.ceph.com
+
+   You would need access to sepia lab for this. If you don't have one, you can
+   request that.
+
+#. You would need to get teuthology commands installed. Run the first set of
+   commands in `Running Your First Test`_. After that, activate the virtual
+   environment where teuthology is installed.
+
+#. Run the ``teuthology-suite`` command::
+
+        teuthology-suite -v -m smithi -c wip-cephfs-shell-feature-x -s fs -p 50 --filter "cephfs-shell"
```

Comment:
Yeah, true. Nice point!

Reply:
Fixed.

Reply:
> Ask the user to check [1] and decide the priority after a --dry-run.

Adding this part to `Testing Priority` itself rather than to this part of the page.

Reply:
> Don't give that 50 there, as everyone would start hundreds of jobs with priority 50

I'll mention 110 since default is 1000, I suppose, which is too much.

Reply:
https://github.com/ceph/ceph/pull/32145